### DISCUSSION TITLE
Way to add to playlist while media is currently playing

### DISCUSSION THREAD

**Discussion by advice2020** [mpv-player/mpv#14702] [Q&A] [answered]
Linux Debian based distro / GTK based Desktop Environment
MPV 0.38

I was wondering if it was possible to add to MPVs built in playlist while media is currently playing?
For example if you are in a file manager and you launch a media item, say a video.
While that video is playing you go back into your file manager and then select another media item and it gets added to play next in the playlist after the current media that is playing is over.

I know this can be done using UMPV set to "append" but I was wondering if there was any actual built in commands within MPV that allow anything like this?  Or any other methods?

Thank You

**Comment by guidocella** [comment, marked as answer]
https://mpv.io/manual/master/#command-loadfile

**Comment by advice2020** [comment]
[Respone removed no longer relevant]
I thought the "loadfile" solution mentioned was something that could be used in command line.

**Reply by guidocella** [reply]
Commands are used in input.conf, scripts and JSON IPC, not as command line arguments.

**Reply by advice2020** [reply]
Thank you for your response, oh I thought this could be used in a command somehow, because then how would I achieve what I was looking to do in a file manager?
So I will probably have to go the route of UMPV as I mentioned?

**Reply by guidocella** [reply]
umpv calls the loadfile command through JSON IPC. If you don't want to use umpv you need to script that yourself.

**Reply by advice2020** [reply]
@guidocella 
Thank you for your response.
No I don't think that is what I am saying, I am okay with using UMPV (I do not know how to script this myself anyways so do not have a choice :)
I was just saying that because I am looking to do this in a file manager, I was thinking it would have to be something in the form of a mpv command.  The only thing I could think of was to have two separate UMPV commands, one set to "replace" and then would need to have one set to "append" to achieve what I was looking to achieve in this post.  I cannot remember if I tried this but I think this should work, just wondering if this was the best way to achieve this.
Thank you again for the help you have been providing with all my questions.

EDIT:
I tried this out using UMPV and it does work if anyone is interested in this in the future.
